src/com/lftechnology/java/training/srijan/basics/Calculator.java

```diff
@@ -0,0 +1,150 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.logging.Logger;
+
+/**
+ * Write a program that calculates percentage and GPA of 5 students. User will
+ * enter the marks of each 5 students from the keyboard. Each of the students
+ * has following attributes: name, class, rollnum, and marks obtained in 5
+ * subjects. Calculate the GPA and show the student name and GPA by
+ * position.[higher GPA should display on top]
+ * 
+ * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+ *
+ */
+
+public class Calculator {
```

Comment:
write proper class name. this class only manipulates subjects .so the class name should relate with subject aswell